### PR TITLE
[AIRFLOW-7106] Cloud data fusion integration - Allow to pass args to start pipeline

### PR DESCRIPTION
**Pull request by malagoli**
---
Issue link: [AIRFLOW-7106](https://issues.apache.org/jira/browse/AIRFLOW-7106)

Make sure to mark the boxes below before creating PR: [x]

- [X ] Description above provides context of the change
- [X ] Unit tests coverage for changes (not needed for documentation changes)
- [ X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
@turbaszek  ? :)

File: airflow/providers/google/cloud/hooks/datafusion.py [hunk 1]
```diff
@@ -397,7 +400,13 @@ def start_pipeline(
...
             "start",
         )
-        response = self._cdap_request(url=url, method="POST")
+
+        if runtime_args != None:
+            args = ast.literal_eval(runtime_args)
+        else:
+            args = None
+
+        response = self._cdap_request(url=url, method="POST", body=args)
```
Comment (turbaszek) [inline airflow/providers/google/cloud/hooks/datafusion.py]: Why pass `runtime_args` as a string? Can't we use `dict`?

Comment (malagoli) [reply airflow/providers/google/cloud/hooks/datafusion.py]: thanks turbaszek, you are right.
I've updated from string to dict, updated also tests

File: tests/conftest.py [hunk 2]
```diff
@@ -131,8 +131,9 @@ def breeze_test_helper(request):
     # Setup test environment for breeze
     home = os.path.expanduser("~")
     airflow_home = os.environ.get("AIRFLOW_HOME") or os.path.join(home, "airflow")
+    print(airflow_home)
 
-    print(f"Home of the user: {home}\nAirflow home {airflow_home}")
+    #print(f"Home of the user: {home}\nAirflow home {airflow_home}")
```
Comment (turbaszek) [inline tests/conftest.py]: This seems to be unrelated change

File: airflow/providers/google/cloud/hooks/datafusion.py [hunk 3]
```diff
@@ -381,6 +382,8 @@ def start_pipeline(
         :type pipeline_name: str
         :param instance_url: Endpoint on which the REST APIs is accessible for the instance.
         :type instance_url: str
+        :param runtime_args: Optional runtime JSON args to be passed to the pipeline
+        :type runtime_args: dict
```
Comment (turbaszek) [inline airflow/providers/google/cloud/hooks/datafusion.py]:
```suggestion
        :type runtime_args: Optional[Dict[str, Any]]
```

File: tests/conftest.py (hunk 2 above)
Comment (malagoli) [reply tests/conftest.py]: correct, my fault. fixed

**Review by potiuk — APPROVED** [review]
LGTM

**Comment by turbaszek** [comment]
It seems that Travis is sad because static code checks are failing. You may find this useful: https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst

**Comment by malagoli** [comment]
fixed also all static checks

**Comment by malagoli** [comment]
@turbaszek all travis checks passed, any change to merge?
thanks
